internal/storage/index/index.go

```diff
@@ -450,33 +450,38 @@ func (idx *index) Inspect() map[string]meta {
 	return entries
 }
 
-func (idx *index) InspectPolicies(ctx context.Context) (map[string]*responsev1.InspectPoliciesResponse_Result, error) {
-	policyIDs, err := idx.ListPolicyIDs(ctx)
-	if err != nil {
-		return nil, fmt.Errorf("failed to list policies: %w", err)
-	}
-
-	if len(policyIDs) == 0 {
-		return nil, nil
-	}
-
+func (idx *index) InspectPolicies(ctx context.Context, file ...string) (map[string]*responsev1.InspectPoliciesResponse_Result, error) {
```

Comment:
There's a bug here. If the user does not explicitly provide any policy IDs to inspect, the result should contain everything in the store. So, you still need to call `ListPolicyIDs` if the `file` argument is empty.